libs/sdk-ui-dashboard/src/presentation/topBar/shareIndicators/lockedStatus/DefaultLockedStatus.tsx

```diff
@@ -0,0 +1,17 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+
+import { ILockedStatusProps } from "./types";
+import { selectSettings, useDashboardSelector } from "../../../../model";
+import { LockedStatusIndicator } from "./LockedStatusIndicator";
+
+/**
+ * @alpha
+ */
+export const DefaultLockedStatus = (props: ILockedStatusProps): JSX.Element | null => {
```

Comment:
The correct typing of functional component is `export const DefaultLockedStatus: React.FC<ILockedStatusProps> = (props) => {`